exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+var greetingStart = "Hello, my name is ";
+var name = "Meron";
```

Comment:
We should not use var despite of the exercise. At the moment we use const and let. Please have a look at this.